types/index.d.ts

```diff
@@ -0,0 +1,142 @@
+// Copyright © 2015, 2017 IBM Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as nano from 'nano';
+import { Request, CoreOptions } from "request";
+
+declare function cloudant(
+    config: cloudant.Configuration | string
+  ): cloudant.ServerScope | cloudant.DatabaseScope;
+
+declare namespace cloudant {
```

Comment:
I'm a bit confused about why this is a `namespace` since Node.js uses modules. Honestly I don't know enough about Node.js packaging and TypeScript to know what is right or not, but I'm going [from this statement](http://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html):
>To reiterate why you shouldn’t try to namespace your module contents, the general idea of namespacing is to provide logical grouping of constructs and to prevent name collisions. Because the module file itself is already a logical grouping, and its top-level name is defined by the code that imports it, it’s unnecessary to use an additional module layer for exported objects.

Again I know next to nothing about TypeScript and have had about 5 minutes with the docs, but I feel like this should be following the pattern of the [module template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) - what am I misunderstanding here?